.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,7 @@
 *Please check each of these after submitting your pull request:*
 
 * [ ] Are you including a `repo_url`? You need to pick the single best GitHub repository for your project, not a GitHub organization.
-* [ ] Is your project open source, and does your project have at least 300 GitHub stars?
+* [ ] Is your project open source?
 * [ ] Has this project been used successfully in the production of a film, television program, animated content or other projects within the scope of the ASWF. Was it designed specifically for the entertainment industry?
```

Comment:
Any hobby project can probably claim some single obscure film where somebody tried it out.  Can I suggest raising the bar just a little here, maybe something like, "Is the project actively used to make films at multiple studios?" 

Also, I think it might be nice to also add

* [ ] Is the project currently and actively maintained (e.g., accepts PRs, replies to issues, has been tested against / builds properly with the compilers for this year's VFX Platform)?

Reply:
By all means @lgritz  - feel free to make the changes directly :-)